components/rag-wrapper/src/rag_wrapper/rag_ops/azure_ai_search_rag_ops.py

```
@@ -50,139 +42,78 @@ def __init__(
-            self.logger.error(
-                f"Failed to initialize Azure AI Search RAG operations: {e}"
-            )
-            raise
 
     async def persist_index(self):
         """No-op as Azure AI Search automatically persists the index."""
         # Index automatically persisted to Azure AI Search service - no action needed
         pass
 
+
     def _get_credentials(self):
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `_get_credentials`

(ANN202)